bundles/org.openhab.binding.draytonwiser/src/main/java/org/openhab/binding/draytonwiser/internal/discovery/DraytonWiserDiscoveryService.java

```diff
@@ -0,0 +1,214 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.draytonwiser.internal.discovery;
+
+import static org.openhab.binding.draytonwiser.internal.DraytonWiserBindingConstants.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.config.discovery.AbstractDiscoveryService;
+import org.eclipse.smarthome.config.discovery.DiscoveryResult;
+import org.eclipse.smarthome.config.discovery.DiscoveryResultBuilder;
+import org.eclipse.smarthome.config.discovery.DiscoveryService;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandlerService;
+import org.openhab.binding.draytonwiser.internal.DraytonWiserRefreshListener;
+import org.openhab.binding.draytonwiser.internal.handler.DraytonWiserPropertyHelper;
+import org.openhab.binding.draytonwiser.internal.handler.HeatHubHandler;
+import org.openhab.binding.draytonwiser.internal.model.DeviceDTO;
+import org.openhab.binding.draytonwiser.internal.model.DraytonWiserDTO;
+import org.openhab.binding.draytonwiser.internal.model.HotWaterDTO;
+import org.openhab.binding.draytonwiser.internal.model.RoomDTO;
+import org.openhab.binding.draytonwiser.internal.model.RoomStatDTO;
+import org.openhab.binding.draytonwiser.internal.model.SmartPlugDTO;
+import org.openhab.binding.draytonwiser.internal.model.SmartValveDTO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link DraytonWiserDiscoveryService} is used to discover devices that are connected to a Heat Hub.
+ *
+ * @author Andrew Schofield - Initial contribution
+ */
+@NonNullByDefault
+public class DraytonWiserDiscoveryService extends AbstractDiscoveryService
+implements DiscoveryService, ThingHandlerService, DraytonWiserRefreshListener {
```

Comment:
fix formatting

Reply:
Presumably to all on one line?

Reply:
the second line should be indented

Reply:
Yeah, I've just run spotless and fixed these 👍